themes/bootstrap3/templates/RecordTab/overdriveholdings.phtml

```diff
@@ -44,16 +45,16 @@
         <?php if (($avail->data->availabilityType ?? 0) == 'AlwaysAvailable'): ?>
           <?=$this->transEsc('od_unlimited')?>
         <?php else: ?>
-          <?=$this->transEsc('od_copies_available', ['%%copies%%' => $avail->data->copiesAvailable . '/' . $avail->data->copiesOwned])?>
+          <?=$this->transEsc('od_copies_available', ['%%copies%%' => $copiesAvailable . '/' . ($avail->data->copiesOwned ?? 0)])?>
         <?php endif; ?>
 
-        <?php if ($avail->data->numberOfHolds > 0): ?>
-          <?='; ' . $this->transEsc('od_waiting', ['holds' => $avail->data->numberOfHolds], null, true)?>
+        <?php if (($numHolds = $avail->data->numberOfHolds ?? 0) > 0): ?>
+          <?='; ' . $this->transEsc('od_waiting', ['holds' => $numHolds], null, true)?>
```

Comment:
Same concern here, I'm not sure that saying zero is better than a clean error, although it's certainly better than the ugly current error.

Reply:
This block should only render if the result is greater than zero. Are you seeing otherwise?